app/jobs/results_recorder_job.rb

```diff
@@ -21,8 +21,7 @@ class ResultsRecorderJob < ApplicationJob
   # @param [String] druid
   # @param [Integer] version
   # @param [String] delivery_class Name of the worker class that performed delivery
-  # @param [String] checksum
-  def perform(druid, version, _delivery_class, _checksum)
+  def perform(druid, version, _delivery_class)
```

Comment:
Right.  Or if we have reached the future now and don't want it, we can remove it, but we would also need to disable queue locking for this job, so that different delivery workers can still queue messages about the same druid/version.

Reply:
I have a slight preference for a longer name that is clearer, but I won't go to the mat for it.